tsconfig.json

```diff
@@ -1,23 +1,23 @@
 {
-	"compilerOptions": {
-		"module": "commonjs",
-		"target": "es2017",
-		"lib": [
-			"es2017",
-			"dom",
-			"dom.iterable"
-		],
-		"sourceMap": true,
-		"experimentalDecorators": false,
-		"strict": true,
-		"baseUrl": ".",
-		"allowJs": true,
-		"outDir": "ts-gen",
-		"paths": {
-			"@electron-internal/*": ["lib"]
-		}
-	},
-	"exclude": [
-		"electron.d.ts"
-	]
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2017",
+    "lib": [
+      "es2017",
+      "dom",
+      "dom.iterable"
+    ],
+    "sourceMap": true,
+    "experimentalDecorators": false,
+    "strict": true,
+    "baseUrl": ".",
+    "allowJs": true,
+    "outDir": "ts-gen",
+    "paths": {
+      "@electron/internal/*": ["lib/*"]
+    }
+  },
+  "exclude": [
```

Comment:
can this be `include` instead of `exclude`?

Reply:
Not easily, we have to explicitly `exclude` this file and this file only

Reply:
Otherwise TS get's multiple-type-declaration errors with the TS defs GN passes in